tutorial/chapter1.md

```diff
@@ -314,6 +310,27 @@ int main(int argc, char *argv[])
     }
 }
 ```
+Once, a quit sequence is pressed by the user join() will be called for our `imageProcessingServiceThread` and sequentially we will shutdown `fileImageStreamingService`, `imageProcessingServices`, we will remove our listeners, and, stop the event loop. 
+
+At this point, it is considered useful to understand the code of the [image_processor_service.cpp](../modules/image_processing/src/image_processor_service.cpp)  / [image_processor_service.h](../modules/image_processing/include/image_processing/image_processor_service.h) 
+
+Follows, a screenshot of the output after the execution of datamux example 1. (After, pressing the termination sequence) 
+
+![datamux example 1 output more details](../images/datamux_example1_output_more.png)
+
+## Image multiplexer example 2
+
+The main difference between the two datamux examples is that the example 2 creates 5 subscribers and 5 listeners in contrast with example 1 that creates only 1 subscriber with 5 listeners. 
+
+This actually means that example 2 is parallelized and that is visible by the output of the two examples as we will see below.
```

Comment:
As mentioned in [kpsr-tutorial](https://github.com/klepsydra-technologies/kpsr-tutorial/blob/main/tutorials/chapter1.md#example-5-the-data-multiplexer), listeners registered in different subscribers will be run in parallel. We will illustrate that fact in the output shown below.